main.yml

```
@@ -611,23 +611,18 @@ openshift:
         default: true
       - id: subscriptions
         title: Subscriptions
+        section: insights
         sub_apps:
-          - id: openshift-sw
-            title: Container Platform
-            group: subscriptions
-            reload: ../subscriptions/openshift-sw/ocp
-          - id: openshift-dedicated
-            title: Dedicated
-            group: subscriptions
-            reload: ../subscriptions/openshift-sw/dedicated
-          - id: opneshift-sw-documentation
-            title: Documentation
-            group: subscriptions
-            navigate: https://access.redhat.com/documentation/en-us/cost_management_service/
-        group: openshift
+            - id: openshift-container
+              title: Container Platform
+            - id: openshift-dedicated
+              title: Dedicated
+            - id: opneshift-sw-documentation
```

I know you didn't change this line, but this would be a great time to fix the `opneshift` typo :)
```suggestion
            - id: openshift-sw-documentation
```